### PR TITLE
add short subchapter with interactive learn git resources

### PR DESCRIPTION
**Pull request by JessicaS11**
### Summary

Adds a short subchapter to describe and show an interactive, visual, browser-based tool called [LearnGitBranching](https://learngitbranching.js.org/) for new (and experienced!) git users.

### List of changes proposed in this PR (pull-request)

- [x] add subchapter
- [x] add figure illustrating the tool
- [ ] solicit feedback on subchapter text
- [ ] get feedback on figure
- [ ] revise and edit

### What should a reviewer concentrate their feedback on?

- [ ] Clarity, grammar, file naming
- [ ] Everything looks ok?

### Acknowledging contributors

- [ ] All contributors to this pull request are already named in the [table of contributors](https://github.com/alan-turing-institute/the-turing-way/blob/master/README.md#contributors) in the README file.
- [x] The following people should be added to the [table of contributors](https://github.com/alan-turing-institute/the-turing-way/blob/master/README.md#contributors) in the README file: @jessicas11

### CODE REVIEW

File: book/website/reproducible-research/vcs/vcs-git-interactive.md [hunk 1]
```diff
@@ -0,0 +1,19 @@
+(rr-vcs-interactive)=
+# Interactive, Visual Git
+
+One of the biggest challenges to getting started with Git can be understanding the file and directory structure and complexity of branches, commits, remotes, etc. and how they interact, particularly on the command line with limited visual representations.
```
Comment (inwaves) [inline book/website/reproducible-research/vcs/vcs-git-interactive.md:4]: Hi! We'll need to substitute the "etc." here with something else to pass our check for Latin phrases. These are useful, but are sometimes more difficult to comprehend than more usual alternatives. There's a bit more info [here](https://the-turing-way.netlify.app/community-handbook/style.html?highlight=style#avoid-latin-abbreviation).

**Review by inwaves — COMMENTED** [review]
Hi @JessicaS11 Thank you for contributing. The PR looks good to me—I left a small comment on line 4. Once that's updated, we can get this merged in.

File: book/website/reproducible-research/vcs/vcs-git-interactive.md (hunk 1 above)
Comment (JessicaS11) [reply book/website/reproducible-research/vcs/vcs-git-interactive.md]: Thanks for the reminder @inwaves!

**Review by inwaves — APPROVED** [review]

**Comment by inwaves** [comment]
@JessicaS11 great! Once the CI build passes, this can be merged.

@all-contributors please add @JessicaS11 for content.

**Pull request merged**